### PR TITLE
docs: mention Consul API gateway in Ingress Controllers page

### PR DESCRIPTION
**Pull request by david-yu**
- Mention Consul API Gateway in Ingress Controllers K8s page
- Remove link to Ambassador API GW learn guide from docs

### CODE REVIEW

File: website/content/docs/k8s/connect/ingress-controllers.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ description: Configuring Ingress Controllers With Consul On Kubernetes
 [Transparent Proxy](/docs/connect/transparent-proxy) mode enabled.
 
 This page describes a general approach for integrating Ingress Controllers with Consul on Kubernetes to secure traffic from the Controller
-to the backend services. This allows Consul to transparently secure traffic from the ingress point through the entire traffic flow of the service.
+to the backend services by deploying sidecars along your Ingress Controller. This allows Consul to transparently secure traffic from the ingress point through the entire traffic flow of the service. If you are looking for a fully supported solution for ingress traffic into Consul Service Mesh, please visit [Consul API Gateway](https://www.consul.io/docs/api-gateway) for instruction on how to install Consul API Gateway along with Consul on Kubernetes.
```
Comment (karl-cardenas-coding) [inline website/content/docs/k8s/connect/ingress-controllers.mdx:15]:
```suggestion
to the backend services by deploying sidecars along with your Ingress Controller. This allows Consul to transparently secure traffic from the ingress point through the entire traffic flow of the service. 

If you are looking for a fully supported solution for ingress traffic into Consul Service Mesh, please visit [Consul API Gateway](https://www.consul.io/docs/api-gateway) for instruction on how to install Consul API Gateway along with Consul on Kubernetes. 
```

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by hc-github-team-consul-core** [comment]
:cherries: If backport labels were added before merging, cherry-picking will start automatically.

To retroactively trigger a backport after merging, add backport labels and re-run https://circleci.com/gh/hashicorp/consul/576381.

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit bad0a6bfe35672e5d929242109038ad4bc86f54e onto `stable-website` succeeded!

**Comment by hc-github-team-consul-core** [comment]
:cherries::white_check_mark: Cherry pick of commit bad0a6bfe35672e5d929242109038ad4bc86f54e onto `release/1.11.x` succeeded!

**Pull request merged**